### PR TITLE
CF-79 Verify migration of floating IPs into pre-created networks.

### PR DESCRIPTION
**Pull request by plomakin**
Fixed generate_load.
Added dst external network.

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module tests.generate_load
devlab/tests/generate_load.py:27: [W0403(relative-import), ] Relative import 'base', should be 'tests.base'
devlab/tests/generate_load.py:28: [W0403(relative-import), ] Relative import 'cleanup', should be 'tests.cleanup'
devlab/tests/generate_load.py:29: [W0403(relative-import), ] Relative import 'config', should be 'tests.config'
devlab/tests/generate_load.py:594: [E1101(no-member), Prerequisites.create_security_group] Instance of 'dict' has no 'id' member
devlab/tests/generate_load.py:668: [E1101(no-member), Prerequisites.create_cinder_volumes] Instance of 'dict' has no 'id' member

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

File: devlab/tests/generate_load.py [hunk 1]
```diff
@@ -956,6 +956,21 @@ def create_network_with_segm_id(self):
...
         self.dst_cloud.create_networks(self.config.dst_networks)
 
+    def create_ext_net_map_yaml(self):
+        src_cloud = Prerequisites(config=self.config,
+                                  configuration_ini=self.configuration_ini)
+        src_ext_nets = [net['name'] for net in self.config.networks
+                        if net.get('router:external')]
+        dst_ext_nets = [net['name'] for net in self.config.dst_networks
+                        if net.get('router:external')]
+        with open(self.config.ext_net_map, "w+") as f:
```
Comment (roman-verchikov) [inline devlab/tests/generate_load.py]: `"w"` should be enough here

File: devlab/tests/generate_load.py [hunk 2]
```diff
@@ -956,6 +956,21 @@ def create_network_with_segm_id(self):
...
+            for src_net in src_ext_nets:
+                for dst_net in dst_ext_nets:
+                    if src_net == dst_net:
+                        network_map = '{0}: {1}'.format(src_cloud.get_net_id(
+                                src_net), self.dst_cloud.get_net_id(dst_net))
+                        f.write(network_map)
```
Comment (roman-verchikov) [inline devlab/tests/generate_load.py]: Perhaps would be more readable with:
```
src_net_id = src_cloud.get_net_id(src_net)
dst_net_id = self.dst_cloud.get_net_id(dst_net)
f.write("{src_net}: {dst_net}".format(src_net=src_net_id, dst_net=dst_net_id))
```

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module tests.generate_load
devlab/tests/generate_load.py:27: [W0403(relative-import), ] Relative import 'base', should be 'tests.base'
devlab/tests/generate_load.py:28: [W0403(relative-import), ] Relative import 'cleanup', should be 'tests.cleanup'
devlab/tests/generate_load.py:29: [W0403(relative-import), ] Relative import 'config', should be 'tests.config'
devlab/tests/generate_load.py:594: [E1101(no-member), Prerequisites.create_security_group] Instance of 'dict' has no 'id' member
devlab/tests/generate_load.py:668: [E1101(no-member), Prerequisites.create_cinder_volumes] Instance of 'dict' has no 'id' member

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Pull request closed without merging**